eo-maven-plugin/src/main/java/org/eolang/maven/AssembleMojo.java

```diff
@@ -169,6 +175,14 @@ public final class AssembleMojo extends SafeMojo {
         defaultValue = "true")
     private boolean failOnError = true;
 
+    /**
+     * Parsed cache directory.
+     * @checkstyle MemberNameCheck (7 lines)
+     */
+    @Parameter(property = "eo.parsed.cache")
```

Comment:
@mximp I suggest either rename it to `eo.cache` and make it equal to `~/.eo`, or make a static constant and don't allow users to change it at all (this is how Maven works, nobody can change the location of `~/.m2` directory).